package.json

```diff
@@ -33,10 +33,10 @@
     "d3": "3.5.17",
     "tangram.cartodb": "cartodb/tangram.cartodb#master",
     "jquery": "2.1.4",
-    "leaflet": "0.7.3",
+    "leaflet": "1.0.3",
     "mustache": "1.1.0",
     "perfect-scrollbar": "git://github.com/CartoDB/perfect-scrollbar.git#master",
-    "torque.js": "CartoDB/torque#master",
+    "torque.js": "CartoDB/torque#leaflet-migration",
```

Comment:
OK, forget about it. I was thinking about the torque version we use in the old editor, but it's an old one 2.11.x, so this should be OK.